polkadot-js/alphanet-types.json

```diff
@@ -67,5 +59,13 @@
     "state_root": "Option<H256>",
     "logs_bloom": "H2048",
     "status_code": "Option<U64>"
+  },
+  "ExitReason": {
+    "_enum": {
+      "ExitSucceed": "bool",
+      "ExitError": "bool",
+      "ExitRevert": "bool",
+      "ExitFatal": "bool"
+    }
```

Comment:
The nested types aren't bool for all of these. For example, see how `ExitSucceed` is defined https://github.com/rust-blockchain/evm/blob/12b6a5e9267304c2c666f4e54935b45541cdb5a7/core/src/error.rs#L67-L78

Reply:
Im still struggling with rust :s
So what is ExitSucced? if it's an enum, what is "Stopped" ?

Reply:
@JoshOrndorff let me know if the updated type is better

Reply:
Yeah, looks good!